modules/swagger-models/src/main/java/io/swagger/models/SecurityRequirement.java

```diff
@@ -1,19 +1,26 @@
 package io.swagger.models;
 
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class SecurityRequirement {
     private String name;
     private List<String> scopes;
+//    private final Map<String, Object> vendorExtensions = new HashMap<String, Object>();
```

Comment:
can't have two @JsonAnyGetter/@JsonAnySetter maps. Being compliant with the primary purpose of this object seems more important than having vendor extensions in this object at this point.